### DISCUSSION TITLE
Unwanted RPM with G-Helper

### DISCUSSION THREAD

**Discussion by AtillaDAMAR** [seerge/g-helper#3318] [Q&A] [answered]
Hi, I have ASUS Tuf A15 Ryzen 7 7735HS with RTX 4060. I use G-Helper for so long and I am happy with it. But I have a problem.
For me, being silent is more important than temperature so I use silent mode in G-Helper. It is great while turbo boost is closed. But when I open turbo boost, fans turn with high RPM and that causes noise. I made a custom fan graph but RPM doesn't work as that graph. What can I do? Btw, I limit my CPU to 30 Watt while using turbo.
Also, I checked VRAM and other component temps and as shown in picture, nothing hit 90 degree.
![unwanted-rpm-with-g-helper-v0-fd3dgb5si8xd1 (1)](https://github.com/user-attachments/assets/a556a92c-b869-4ebb-b8e5-a70dc15e872a)

**Comment by seerge** [comment, marked as answer]
@Lejax536 hello, fans are [controlled by firmware / BIOS](https://github.com/seerge/g-helper/wiki/FAQ#how-does-g-helper-control-my-fan-speeds) even if you have set a custom fan curve. And if BIOS believes that device is too hot (78C on your screenshot) - it will still run fans as it wants, it's normal.

If you really want to override BIOS behavior by some reason - you can try [Experimental build with a manual fan control](https://github.com/seerge/g-helper/discussions/2272)

**Reply by AtillaDAMAR** [reply]
Thank you so much for your fast reply. Do you think 78C is too hot or is just BIOS read as that? And do you suggest trying the experimental version? Can I turn back if I don't like it?

**Reply by AtillaDAMAR** [reply]
> Thank you so much for your fast reply. Do you think 78C is too hot or is just BIOS read as that? And do you suggest trying the experimental version? Can I turn back if I don't like it?

Also, is it possible to add custom turbo boost frequency? I know Windows has a feature like that and I used it reduce the temperature before using G-Helper. But it would be so good in G-Helper.

**Reply by seerge** [reply]
I assume 78C is hot enough for BIOS on your model to force fans. Experimental build sets fan speeds only while it's running, if you quit it or even uncheck Apply Custom Fan curve - you are back to default BIOS controls.

You can still limit CPU frequency in windows power plan same as you did before it worked for you, G-Helper won't override that as that's just a setting in power plan.

**Reply by AtillaDAMAR** [reply]
Thank you so much for all your effort :)
